package.json

```diff
@@ -13,11 +13,14 @@
   },
   "dependencies": {
     "argon2": "^0.29.1",
+    "bootstrap": "^5.2.2",
```

Comment:
We can `npm uninstall bootstrap` since it is already linked in the HTML document.